src/solvers/mp/RCPricer.cpp

```diff
@@ -33,26 +33,26 @@ RotationPricer::RotationPricer(MasterProblem* master, const char* name, SolverPa
 }
 
 /* Destructs the pricer object. */
-RotationPricer::~RotationPricer() {
-	for(pair<const Contract*, SubProblem*> p: subProblems_)
+RCPricer::~RCPricer() {
+	for(auto& p: subProblems_)
 		delete p.second;
 }
 
-void RotationPricer::initPricerParameters(SolverParam param){
+void RCPricer::initPricerParameters(const SolverParam& param){
 	// Here: doing a little check to be sure that secondchance is activated only when the parameters are different...
```

Comment:
I guess this check should generate at least a warning when second chance is deactivated due to wrong choice of default parameter...